.test-infra/jenkins/job_PostCommit_Python_CrossLanguage_Gcp_Direct.groovy

```diff
@@ -44,7 +44,6 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Python_Xlang_Gcp_Direct',
           rootBuildScriptDir(commonJobProperties.checkoutDir)
           tasks(":sdks:python:test-suites:direct:gcpCrossLanguagePostCommit")
           commonJobProperties.setGradleSwitches(delegate)
-          switches("-PuseWheelDistribution")
```

Comment:
Did you mean to leave this out?

Reply:
This flag was not effective at the first place. It's Direct runner, and the flag only affects dataflow runner